sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/PhoneNumberIdentifier.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.common;
+
+import com.azure.android.core.util.CoreUtil;
+
+/**
+ * Communication identifier for Communication Services representing a PhoneNumber
+ */
+public class PhoneNumberIdentifier extends CommunicationIdentifier {
+
+    private final String phoneNumber;
+    private String id;
+
+    /**
+     * Creates a PhoneNumber object
+     *
+     * @param phoneNumber the string identifier representing the PhoneNumber
+     * @throws IllegalArgumentException thrown if phoneNumber parameter fail the validation.
+     */
+    public PhoneNumberIdentifier(String phoneNumber) {
+        if (CoreUtil.isNullOrEmpty(phoneNumber)) {
+            throw new IllegalArgumentException("The initialization parameter [phoneNumber] cannot be null or empty.");
+        }
+        this.phoneNumber = phoneNumber;
+    }
+
+    /**
+     * Gets the string identifier representing the object identity
+     *
+     * @return the string identifier representing the object identity
+     */
+    public String getPhoneNumber() {
```

Comment:
Let's copy over the verbatim comments from C#. It's important that we explain that the number is in E.164 format.

> The phone number in E.164 format.